src/utils/lib/hasOnlyTheseExternalPayments.js

```diff
@@ -0,0 +1,38 @@
+function hasOnlyTheseExternalPayments (objUnit, arrExpectedOutputs) {
```

Comment:
Could you please provide a some description for this method in `Utils` section of README?

Reply:
Sorry, I didn't notice the lint script.
I added a description in README.